lib/FetchEnvs.js

```diff
@@ -27,6 +27,17 @@ const KIND_MAP = new Map([
   ['configMapKeyRef', 'ConfigMap']
 ]);
 
+const LRU = require('lru-cache');
+const LruOptions = {
+  maxSize: 100000, // the max cache size
+  sizeCalculation: (r) => { return( JSON.stringify(r).length ); }, // how to determine the size of a resource added to the cache
+  ttl: 1000 * 60 * 3,  // max time to cache (LRU does not directly enforce, but maxSize will eventually push them out)
+  updateAgeOnGet: false,  // Don't update ttl when an item is retrieved from cache
+  updateAgeOnHas: false,  // Don't update ttl when an item is checked in cache
```

Comment:
Thinking you want these to be true.

Reply:
No, I think we want these to be 'false' so that we can guarantee the cache clears periodically.  If 'true', then if a configmap is retrieved frequently it would _never_ be cleared from the cache unless something like RRM has a watch on it and clears it explicitly.  With 'false', we can be sure a configmap/secret/whatever will not be cached forever and updates _will_ eventually be retrieved and used.  'false' is actually the default for `lru-cache`, but I wanted to spell it out with the comments so it was obvious this was intentional.